xml/System.Runtime.Serialization/ImportOptions.xml

```diff
@@ -107,7 +107,7 @@
       <Docs>
         <summary>Gets or sets a data contract surrogate that can be used to modify the code generated during an import operation.</summary>
         <value>An implementation of the <see cref="T:System.Runtime.Serialization.IDataContractSurrogate" /> interface that handles schema import.</value>
-        <remarks>To be added.</remarks>
+        <remarks>The interface type for this option is <see cref="T:System.Runtime.Serialization.ISerializationSurrogateProvider"/>, but to take full advantage of the imported code modification abilities, using an <see cref="T:System.Runtime.Serialization.ISerializationSurrogateProvider2" /> that also implements <see cref="T:System.Runtime.Serialization.ISerializationCodeDomSurrogateProvider" /> is recommended.</remarks>
```

Comment:
Both types exists, not sure why we're getting warnings:

[src/libraries/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/ISerializationSurrogateProvider2.cs#L15](https://github.com/dotnet/runtime/blob/802bbbdb3232912c6343db7a31e50ed63a59dc46/src/libraries/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/ISerializationSurrogateProvider2.cs#L15)

[src/libraries/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/ISerializationSurrogateProvider.cs#L12](https://github.com/dotnet/runtime/blob/802bbbdb3232912c6343db7a31e50ed63a59dc46/src/libraries/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/ISerializationSurrogateProvider.cs#L12)

Reply:
I see the problem: These APIs got added 11 days ago. So the remarks belong to 8.0. I'll remove them.

https://github.com/dotnet/runtime/blob/release/7.0-preview7/src/libraries/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/ISerializationSurrogateProvider2.cs

Reply:
```suggestion
        <remarks>To be added.</remarks>
```

Reply:
For Serialization, the types were added in 7.0-RC1. Not sure why they would be in a -preview7 branch of documents, but they should definitely be there for the final release of 7.0.

Reply:
Okay, thanks for letting me know. The MS Docs API surface right now is targeting Preview7 (the latest released preview version).

After we release RC1 and update the MS Docs APi surface, I'll port the documentation again and these remarks will show up in the PR again, but without CI warnings because the types will be found this time.